src/utils/Auth.js

```diff
@@ -51,5 +7,9 @@ export const verifyToken = (token) => {
       "Content-Type": "application/json",
       authorization: `Bearer ${token}`,
     },
+  }).then((res) => {
+    if (res.ok) {
+      return res.json();
+    }
```

Comment:
Please, import `checkResponse` or `request` from `MainApi.js` not to duplicate the `ok` checking

Reply:
Done